script/install-libseccomp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eux -o pipefail
+
+VERSION=2.5.0
+MD5SUM=463b688bf7d227325b5a465b6bdc3ec4
```

Comment:
nit: SHA256SUM is considered to be more preferable today

Reply:
Oh, will update to check for that

Reply:
libseccomp publishes the .SHA256SUM file on releases, so I've updated this to download the .tar.gz and the .SHA256SUM file and verify it matches.

If you prefer, I can store the SHA256 in the script instead, though.

And as a minot change, updated this to use v2.5.1 too.

Reply:
Well, I've updated it to have the expected SHA256SUM in the script. Even though we are running on github and downloading from github, is not much of overhead to have the SUM on the script and update it on updates. libseccomp is not something that is released super often :)